app/script/directives/product.list.directive.js

```diff
@@ -0,0 +1,21 @@
+(function () {
```

Comment:
please rename this file to something like sortable.list.directive, since we will use it for other data than products as well